evergreen/config.yml

```diff
@@ -666,6 +695,101 @@ tasks:
       test_filter: ObjectStoreTests
       verbose_test_output: true
 
+- name: process_coverage_data
+  display_name: Process Coverage Data
+  tags: [ "for_pull_requests" ]
+  exec_timeout_secs: 1800
+  commands:
+  - command: shell.exec
+    params:
+      working_dir: realm-core
+      shell: bash
+      env:
+        COVERALLS_TOKEN: "${coveralls_token}"
```

Comment:
How did you get this token into the evergreen environment? Will we need to update it eventually?

Reply:
The value of this comes from the coveralls UI for the realm-core repo. I don't know if we'll have to change it, but if you can merge PRs into the realm-core repo you can get whatever the current value is from coveralls.io. As for how I got it into the evergreen UI, it's a project variable configured in the spruce UI - you can read more about that here https://docs.devprod.prod.corp.mongodb.com/evergreen/Configure-a-Project/Project-and-Distro-Settings#project-settings.